exporter/exporterhelper/internal/consumers.go

```diff
@@ -0,0 +1,47 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package internal // import "go.opentelemetry.io/collector/exporter/exporterhelper/internal"
+
+import "sync"
+
+type Consumers struct {
+	queue        Queue
+	numConsumers int
+	callback     func(item QueueRequest)
+	stopWG       sync.WaitGroup
+}
+
+func NewConsumers(q Queue, numConsumers int, callback func(item QueueRequest)) *Consumers {
+	return &Consumers{
+		queue:        q,
+		numConsumers: numConsumers,
+		callback:     callback,
+		stopWG:       sync.WaitGroup{},
+	}
+}
+
+func (c *Consumers) Start() {
```

Comment:
Comment that ensures all consumers are started.

Reply:
Done